src/components/Widgets/Checkbox.jsx

```diff
@@ -34,10 +34,10 @@ const StyledCheckbox = styled.input`
   position: relative;
   margin: 0;
   outline: none !important;
-  -webkit-appearance: none;
```

Comment:
Heads up: we lost styling in Firefox here (I'm only checking Chrome and Firefox atm). Although not a valid regular css key, styled-components resolves this by accepting `appearance: none;` and adding the appropriate prefixes. I'll take care of this.